src/modules/Pam.rb

```diff
@@ -135,11 +136,21 @@ def Set(mod, set)
       set ? Add(mod) : Remove(mod)
     end
 
+    # Check if the system runs a dual PAM stack (64 and 32 bits)
+    # @return true if system runs dual PAM stack
+    def DualStack
+      if FileUtils.Exists("/lib/security") && FileUtils.Exists("/lib/libpam.so.0")
```

Comment:
Sorry for being nitpicking but I don't see the effective difference between these four lines of code:

```ruby
def somehing
  if condition1 && condition2
    return true
  end
  false
end
```

and a more readable one-liner like:

```ruby
def somehing
  condition1 && condition2
end
```

And looks it's not only me, Rubocop is also complaining about that code.